_docs/tasks/security/mutual-tls.md

```diff
@@ -62,6 +65,13 @@ For example, after starting the [Bookinfo]({{home}}/docs/guides/bookinfo.html)
 sample application you can ssh into the Envoy container of `productpage` service,
 and send request to other services by curl.
 
+> By default, istio proxy image does not have `curl` installed. To try this
```

Comment:
It could be misleading, currently --debug flag is turned on in bookinfo demo: https://screenshot.googleplex.com/cRRdeeJWRjt, so chances are many customers don't have this issue at all and put the curl warning here is unnecessary. And please refer to line 103-114 in this file for more info. I feel that's more accurate (customer don't have to redeploy if they already have curl installed)

Reply:
I lost track why I made this change in the first place. Reverted to origin.